skins/10h16/web/_styles/styles.css

```diff
@@ -278,6 +278,11 @@ button.back-button:active { background: #cccccc; }
     -moz-border-radius: 2px;
 }
 
+.input-container .ui-selectmenu-button {
+    width: 350px !important;
```

Comment:
Ok, I see why you use `!important` here - the width is set for the element and using `!important` is the only way to override the element style. But I'd prefer if we found some other way to affect the width of the element, let's have a look at `10h16/web/_js/spenden.wikimedia.de`, there is some width trickery going on there. Let's try to understand what it does, if it's still needed and if we can improve it.

as it is, this selector breaks all the other forms where a select is used (eg.g donation and membership address form).